barretenberg/cpp/src/barretenberg/benchmark/widgets_bench/widget.bench.cpp

```diff
@@ -24,6 +41,11 @@ BasicPlonkKeyAndTranscript get_plonk_key_and_transcript()
     auto builder = typename plonk::UltraComposer::CircuitBuilder();
     bench_utils::generate_basic_arithmetic_circuit(builder, 80);
```

Comment:
Pretty confused about what's going on here. Why is this 80? This is creating a circuit with 2^80 gates, no? The input was intended to be log_2 number of gates and as far as I can tell that's still how it works. Is this not being used as I think?

Reply:
Lol it's overflowing: $2^{80} = 2^{16} \mod 2^{32}$. I printed `passes` in the utility function body to double check and indeed it is equal to $2^{16-2}-4$. I'll just set 80 to 16.